src/Kronometer.cs

```diff
@@ -398,6 +398,11 @@ public virtual string PrintDateDeltaCompact(double time, bool includeTime, bool
             return stringBuilder.ToStringAndRelease();
         }
 
+        public virtual string PrintDateDeltaCompact(double time, bool includeTime, bool includeSeconds, bool useAbs, int uselessint)
```

Comment:
hi @R-T-B , thanks for the PR

are you sure the int is actually useless?

Reply:
For our purpose?s Yes.  To squad?  No idea.